Sources/Pagination/Model+Paginatable.swift

```diff
@@ -12,10 +12,10 @@ import Vapor
 extension Model where Self: Paginatable, Self: Content {
     /// Returns a paginated response on `.all()` entities
     /// using page number from the request data
-    public static func paginate(for req: Request, key: String = Pagination.defaultPageKey, perKey: String = Pagination.defaultPagePerKey, _ sorts: [QuerySort] = Self.defaultPageSorts) throws -> Future<Page<Self>> {
+    public static func paginate(for req: Request, pageKey: String = Pagination.defaultPageKey, perKey: String = Pagination.defaultPagePerKey, _ sorts: [QuerySort] = Self.defaultPageSorts) throws -> Future<Page<Self>> {
         return try Self.query(on: req).paginate(
             for: req,
-            key: key,
+            pageKey: pageKey,
             perKey: perKey,
```

Comment:
Good idea. While we're on the topic, `perKey` could be any one of these options, IMO:
1. `perKey`
 - It could be left as-is since `per` comes with the domain of paging — you sort of already know what `per` means in the context. Though, it could be more explicit.
2. `perPageKey`
 - By changing to `perPageKey` there is no ambiguity.
3. `sizeKey`
 - Since the [default key](https://github.com/vapor-community/pagination/blob/c4d407cd7934ad34e8393af906e44a00c8890184/Sources/Pagination/Pagination.swift#L9) has a value of `size`, I think this option actually increases clarity and consistency across the board.
4. `pageSizeKey`
 - All of the benefits of options 2 & 3. It would remove ambiguity (`size` of...?) and it would more closely resemble the actually key found in the query parameters. On that note, however, it might be worth it to consider changing the value of the [default key](https://github.com/vapor-community/pagination/blob/c4d407cd7934ad34e8393af906e44a00c8890184/Sources/Pagination/Pagination.swift#L9) to `pageSize`. This would add the benefit of disambiguating the `size` query parameter as well as freeing up the parameter to be used by the API developer however they might want (they could, for example, want to use `size` to return a certain size image) while tying the parameter directly to the paging domain.

Option 4 would be my first choice because of all of the added benefits. It would require the most changes, but those could implemented fairly quickly and without much friction. It would be best to make the changes sooner rather than later since client-side changes would have to be made as well. Luckily, this only a few days old!

Reply:
Option 4 sounds better. It'll clear up thee naming convention to better match the industry standard, from what you've mentioned as well as clarity around everything. 

So just to summarize:
The naming for the parameter name as well as the value for size here https://github.com/vapor-community/pagination/blob/c4d407cd7934ad34e8393af906e44a00c8890184/Sources/Pagination/Pagination.swift#L9 will change from `perKey` to `perPageKey` for better descriptiveness, and then `"size"` will change to `"per"`. The only change for the request and response would be minuscule at this point. 
`?page=1&size=10` -> `?page=1&per=10`

I've come across some pagination responses too where per is also named amount. I'd like to follow the best practice or the standard for this naming. 

Go for it and rename, and make everything a bit more descriptive 👍

Reply:
Awesome! I like the changes. I'll add them to the PR. Stay tuned...

Reply:
@anthonycastelli I've made those changes.